custom/icds_reports/migrations/0103_aggregateccsrecordcomplementaryfeedingforms_supervisor_id.py

```diff
@@ -9,7 +9,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('icds_reports', '0101_add_supervisor_id'),
+        ('icds_reports', '0102_drop_update_aggregate_awc_data'),
```

Comment:
This is also being addressed in https://github.com/dimagi/commcare-hq/pull/23543/ since it's more urgent